doc/book/admin/monitoring/getting_started.rst

```diff
@@ -0,0 +1,105 @@
+..  _monitoring-getting_started:
+
+Getting started with monitoring
+===============================
+
+Example on GitHub: `sharded_cluster_crud_metrics <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/sharding/instances.enabled/sharded_cluster_crud_metrics>`_
+
+Tarantool allows you to configure and expose its :ref:`metrics <metrics-reference>` using a :ref:`YAML configuration <configuration>`.
+You can also use the :ref:`metrics <metrics-api_reference>` module to create and collect custom metrics.
```

Comment:
```suggestion
You can also use the built-in :ref:`metrics <metrics-api_reference>` module to create and collect custom metrics.
```
This addition may be helpful for users of previous versions where metrics wasn't on board.